SQL/CreateDatabase.sql

```diff
@@ -36,3 +36,8 @@ CREATE TABLE [LoadForecastValue] (
   CONSTRAINT [PK_LoadForecastValue] PRIMARY KEY ([Forecast], [Horizon]),
   CONSTRAINT [FK_LoadForecastValue_LoadForecast] FOREIGN KEY ([Forecast]) REFERENCES [LoadForecast]([ID])
 );
+GO
+
```

Comment:
Do we want a comment here that they should swap these for their actual operating area names?

Reply:
Yep